BootloaderCommonPkg/Include/Library/BootloaderCommonLib.h

```diff
@@ -72,6 +72,15 @@ typedef enum {
   LOADER_STAGE_PAYLOAD
 } LOADER_STAGE;
 
+//
+// Enum type for SMM rebase enabling
+//
+typedef enum {
+  SMM_REBASE_DISABLE = 0x00,
+  SMM_REBASE_ENABLE,
+  SMM_REBASE_ENABLE_S3
```

Comment:
Change it to  SMM_REBASE_ENABLE_ON_S3_RESUME_ONLY to make it more clear

Reply:
Changed it in new patch.